zerver/models.py

```diff
@@ -1113,6 +1117,17 @@ def content_has_link(content):
         # type: (text_type) -> bool
         return 'http://' in content or 'https://' in content or '/user_uploads' in content
 
+    @staticmethod
+    def is_status_message(content, rendered_content):
+        # type: (text_type, text_type) -> bool
+        """
+        Returns True if content and rendered_content are from 'me_message'
+        """
+        if content.startswith('/me ') and '\n' not in content:
+            if rendered_content.startswith('<p>') and rendered_content.endswith('</p>'):
+                return True
+        return False
+
```

Comment:
Can you split out adding this function as a small commit before the first commit?  Then we can merge that first cleanup commit now, while we iterate on getting the migrations right.